### PR TITLE
Reporting pipeline sierra

### PR DESCRIPTION
**Pull request by harrisonpim**
### What is this PR trying to achieve?
adds sierra bibs to the reporting pipeline

### Who is this change for?
📄 📊

### CODE REVIEW

File: reporting/sierra_transformer/src/transform.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    # unpack material types
+    if "materialType" in bib_record:
+        bib_record["materialType"] = bib_record["materialType"]["code"]
+
+    # unpack orders
+    if "orders" in bib_record:
+        for order in bib_record["orders"]:
+            bib_record["order_locations"] = order["location"]
+            bib_record["order_dates"] = order["date"]
```
Comment (alexwlchan) [inline reporting/sierra_transformer/src/transform.py]: I might be reading this wrong (it’s late), but won’t this discard all but the first order?

You repeatedly overwrite the same `order_{locations, dates}` key, so only the last order will be preserved.

File: reporting/terraform/main.tf [hunk 2]
```diff
@@ -30,6 +31,26 @@ module "lambda_miro_inventory_transformer" {
...
 
+  vhs_read_policy       = "${local.miro_inventory_vhs_read_policy}"
   trigger_topic_arn     = "${local.miro_inventory_topic_arn}"
   error_alarm_topic_arn = "${local.lambda_error_alarm_arn}"
 }
+
+module "lambda_sierra_transformer" {
+  source = "./reporting_lambda"
+
+  name        = "sierra_transformer"
```
Comment (alexwlchan) [inline reporting/terraform/main.tf]: Minor: I'd like us to be better about namespacing our applications. We already have a “sierra_transformer” in the catalogue pipeline – maybe all these Lambdas could have their name prefixed with “reporting-”, or something?

(I checked the module, but couldn’t see anything doing this.)

Comment (mklander) [reply reporting/terraform/main.tf]: 👍yes we also talked about this

**Comment by harrisonpim** [comment]
closing this PR, as its contents are now encompassed by #3110

**Pull request closed without merging**